app/models/spree/gateway/khalti.rb

```diff
@@ -0,0 +1,33 @@
+class Spree::Gateway::Khalti < Spree::Gateway
+  attr_accessor :server, :test_mode
```

Comment:
lets set default value of server to `https://khalti.com` so that we can reuse it for api call